src/main/java/com/financetracker/server/controller/UserController.java

```diff
@@ -22,12 +22,12 @@
 
     @PostMapping("/sign-up")
     public ResponseEntity<?> signUp(@RequestBody User user) {
-        try{
+        try {
             userService.registerUser(user);
         } catch (UserException e){
             LOGGER.error("Sign up failed, error : " + e.getMessage());
         }
-        return new ResponseEntity<>("Creating user failed", HttpStatus.BAD_REQUEST);
+        return new ResponseEntity<>("Creating user succesfully", HttpStatus.BAD_REQUEST);
```

Comment:
toto nebol najšťastnejší fix, teraz vracias successful v oboch prípadoch  a dokonca oba so zlým http codom...ale beriem to na seba, opravoval si môj bug

Reply:
je to aj moja vina, nevsimol som si, ze to vracia zly kod

Reply:
fixnute